web.core/licenseinfo.xml

```diff
@@ -0,0 +1,90 @@
+        <file>src/org/netbeans/modules/web/core/palette/items/resources/Insert16.png</file>
+        <file>src/org/netbeans/modules/web/core/palette/items/resources/Choose16.png</file>
+        <file>src/org/netbeans/modules/web/core/palette/items/resources/ForEach32.png</file>
+        <file>src/org/netbeans/modules/web/core/palette/items/resources/Report32.png</file>
+        <file>src/org/netbeans/modules/web/core/palette/items/resources/Update32.png</file>
+        <file>src/org/netbeans/modules/web/core/palette/items/resources/Delete32.png</file>
+        <file>src/org/netbeans/modules/web/core/palette/items/resources/SetProperty32.png</file>
+        <file>src/org/netbeans/modules/web/core/palette/items/resources/Query32.png</file>
+        <file>src/org/netbeans/modules/web/core/palette/items/resources/Choose32.png</file>
+        <file>src/org/netbeans/modules/web/core/palette/items/resources/Insert32.png</file>
+        <license ref="Apache-2.0-ASF" />
+        <comment type="COMMENT_UNSUPPORTED" />
+    </fileset>
+    <fileset>
+        <file>src/org/netbeans/modules/web/core/resources/templates/AdvancedFilter.template</file>
+        <file>src/org/netbeans/modules/web/core/resources/templates/BlankTagLibrary.template</file>
+        <file>src/org/netbeans/modules/web/core/resources/templates/BlankTagLibrary_1_2.template</file>
+        <file>src/org/netbeans/modules/web/core/resources/templates/BlankTagLibrary_2_0.template</file>
+        <file>src/org/netbeans/modules/web/core/resources/templates/BodyTagHandler.template</file>
+        <file>src/org/netbeans/modules/web/core/resources/templates/JSP.template</file>
+        <file>src/org/netbeans/modules/web/core/resources/templates/JSPDoc.template</file>
+        <file>src/org/netbeans/modules/web/core/resources/templates/JSPDocX.template</file>
+        <file>src/org/netbeans/modules/web/core/resources/templates/JSPF.template</file>
+        <file>src/org/netbeans/modules/web/core/resources/templates/Servlet.template</file>
+        <file>src/org/netbeans/modules/web/core/resources/templates/ServletListener.template</file>
+        <file>src/org/netbeans/modules/web/core/resources/templates/SimpleFilter.template</file>
+        <file>src/org/netbeans/modules/web/core/resources/templates/SimpleTagHandler.template</file>
+        <file>src/org/netbeans/modules/web/core/resources/templates/TagFile.template</file>
+        <file>src/org/netbeans/modules/web/core/resources/templates/TagFileF.template</file>
+        <file>src/org/netbeans/modules/web/core/resources/templates/TagFileFX.template</file>
+        <file>src/org/netbeans/modules/web/core/resources/templates/TagFileX.template</file>
+        <file>src/org/netbeans/modules/web/core/resources/templates/WebModule.template</file>
+
+        <file>src/org/netbeans/modules/web/core/resources/web-jsptaglibrary_1_1.dtd</file>
+        <file>src/org/netbeans/modules/web/core/resources/web-jsptaglibrary_1_2.dtd</file>
```

Comment:
I would not add it here (they are not templates) instead I would replace the files with the version from tomcat:
http://svn.apache.org/repos/asf/tomcat/tc8.5.x/tags/TOMCAT_8_5_33/java/javax/servlet/resources/web-jsptaglibrary_1_1.dtd
http://svn.apache.org/repos/asf/tomcat/tc8.5.x/tags/TOMCAT_8_5_33/java/javax/servlet/resources/web-jsptaglibrary_1_2.dtd
Both are ALv2 and so perfect for us. It would be good to document the source in the commit message.

Reply:
Thanks @matthiasblaesing 
[1] https://issues.apache.org/jira/browse/NETBEANS-1166